test/e2e/util.go

```diff
@@ -1067,6 +1081,7 @@ func RunRC(config RCConfig) error {
 				running++
 				for _, v := range FailedContainers(p) {
 					failedContainers = failedContainers + v.restarts
+					containerRestartNodes.Insert(p.Spec.NodeName)
```

Comment:
maybe store node name *and* container name, and print both in dumpNodeDebugInfo() ?